### PR TITLE
[DOC-10994] Document --tenant-name-scope flag of cockroach cert create-client

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10994] Document --tenant-name-scope flag of cockroach cert create-client

v24.3 product change

### CODE REVIEW

File: src/current/v24.3/cockroach-cert.md [hunk 1]
```diff
@@ -169,6 +169,14 @@ Flag | Description
 
 {% include {{ page.version.version }}/misc/logging-defaults.md %}
 
+### `create-client`
+
+The `cockroach cert create-client` subcommand supports the following flags:
+
+Flag | Description
+-----|-----------
+`--tenant-name-scope` | {% include_cached new-in.html version="v24.3" %} Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.
```
Comment (mikeCRL) [inline src/current/v24.3/cockroach-cert.md:178]: The 'new in' is 'breaking' the table. Renders 'fine', just not as expected—with everything after 'new in' showing as a new paragraph after the table. Check out the preview. Can you think of any good workaround?

Being rendered like this. I assume the 'span' tag from the include is getting in the way, but I'm not sure. Maybe converting this to an HTML table is the simplest solution for now.

```
<table><thead>
<tr>
<th>Flag</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td><code>--tenant-name-scope</code></td>
<td><span class="version-tag">New in v24.3:</span></td>
</tr>
</tbody></table>

<p>Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.</p>
```

**Review by mikeCRL — CHANGES_REQUESTED** [review]

File: src/current/v24.3/cockroach-cert.md (hunk 1 above)
Comment (mdlinville) [inline src/current/v24.3/cockroach-cert.md:178]:
```suggestion
`--tenant-name-scope` | {{ {% include_cached new-in.html version="v24.3" %} | strip }} Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.
```

Comment (mdlinville) [reply src/current/v24.3/cockroach-cert.md]: Let's see if this does it.

File: src/current/v24.3/cockroach-cert.md [hunk 2]
```diff
@@ -169,6 +169,14 @@ Flag | Description
 
 {% include {{ page.version.version }}/misc/logging-defaults.md %}
 
+### `create-client`
+
+The `cockroach cert create-client` subcommand supports the following flags:
+
+Flag | Description
+-----|-----------
+`--tenant-name-scope` | {{ {% include_cached new-in.html version="v24.3" %} | strip }} Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.
```
Comment (mdlinville) [inline src/current/v24.3/cockroach-cert.md:178]:
```suggestion
`--tenant-name-scope` | Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.
```
I'll just remove the new-in here.

**Review by mikeCRL — APPROVED** [review]

**Pull request merged**